functional_tests/mocked/conftest.py

```diff
@@ -0,0 +1,31 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+
+import socket
+
+import pytest
+
+from sdcm.utils.aws_region import AwsRegion
+
+
+AWS_REGION = "us-east-1"
+
+
+@pytest.fixture(scope="session", autouse=True)
+def check_aws_mock():
+    assert socket.gethostbyname("aws-mock.itself") == socket.gethostbyname(f"ec2.{AWS_REGION}.amazonaws.com")
```

Comment:
let's catch gaierror and assert with proper error message os it will be clear to the person who looks on the test result why it is failed
```
In [3]: socket.gethostbyname("aws-mock.itself")
-------------------------------------------------------
gaierror              Traceback (most recent call last)
<ipython-input-3-29db360024dc> in <module>
----> 1 socket.gethostbyname("aws-mock.itself")

gaierror: [Errno -2] Name or service not known

```

Reply:
done